playground/src/email-loader/email-loader.js

```diff
@@ -21,31 +26,29 @@ export function createEmailLoader(editor) {
 class EmailLoader {
   constructor(editor) {
     this.editor = editor;
-  }
 
-  async loadEmailFromFile() {
-    const files = await new Promise((resolve) => {
-      const dialog = document.createElement('input');
-      dialog.setAttribute('type', 'file');
-      dialog.setAttribute('accept', '.eml');
-      dialog.addEventListener('change', () => resolve(dialog.files));
-      dialog.click();
+    events.subscribe(FileUpload.EVENT_FILE_UPLOADED, (file) => {
```

Comment:
If anytime soon there is more than one file upload field in the playground the email loader would try to load and parse every uploaded file. Needs to be decoupled.